### DISCUSSION TITLE
Prevent columns with name "index"?

### DISCUSSION THREAD

**Discussion by afonari** [sqlalchemy/sqlalchemy#6029] [Usage Questions] [answered]
**Is your feature request related to a problem? Please describe.**
Have a table in sqlite DB with a column called "index", tried
```sql
SELECT path
FROM
   structures
   INNER JOIN files ON files.index = structures.file_id
```

it failed.

**Describe the solution you'd like**
Renamed "index" to "id" it worked.

I'm a total noob in SQL. Thanks for the code!

**Comment by zzzeek** [comment, marked as answer]
hi there -

"index" is a reserved word so if you are writing a literal SQL string, you have to quote it:

```
SELECT path
FROM
   structures
   INNER JOIN files ON files."index" = structures.file_id

```

**Reply by afonari** [reply]
Ah, OK, thank! I know there is probably something like this.
